download_data.py

```diff
@@ -0,0 +1,43 @@
+import os
+import argparse
+import requests
+import tarfile
+import requests
+import os
+def download_text_file(url, file_path):
+    response = requests.get(url)
+    if response.status_code == 200:
+        with open(file_path, 'w') as file:
+            file.write(response.text)
+    else:
+        print(f"Failed to download text file. Status code: {response.status_code}")
+
+def read_text_file(file_path):
+    with open(file_path, 'r') as file:
+        contents = file.read()
+        return contents
+    
+
+
+
+def main(args):
+
+    # Construct the URL to download 
+    url = 'https://raw.githubusercontent.com/djones1040/Foundation_DR1/master/Foundation_DR1/'
```

Comment:
Is this url gonna be permanently accessible? Is this the official source of data?